petab/version.py

```diff
@@ -1 +1,2 @@
+"""PEtab version"""
```

Comment:
setup.py might read in this file (didn't check), in that case automatic versioning might break.

Reply:
it thought it `eval`s it. will double check. was to satisfy pylint.

Reply:
`exec(read(version_file))`, so it's safe